config/client_config/dc/system/config/templates/features/enroll_app/hbx_admin_config.yml

```diff
@@ -13,3 +13,5 @@ registry:
           is_enabled: true
         - key: :inbox_tab
           is_enabled: true
+        - key: :aca_individual_market_feature
```

Comment:
I missed this new item on my first pass through, we don't need this new config item. We're already setting IVL market on and off elsewhere. Use the existing check instead: `EnrollRegistry.feature_enabled?(:aca_individual_market)`.